### ISSUE TITLE
Chi symbol displays as <U+03C7>² Tests when Knit

### ISSUE THREAD

**Issue by BobMuenchen** [jamovi/jmv#198]
When running contTables in RStudio, the Chi symbol in Chi-Squared shows up fine in the notebook view, but when it knits to HTML or Word, it shows up as <U+03C7>². Not sure where this can be fixed as it could be RStudio, MiKTeX, Pandoc, as well as jmv.

**Comment by jonathon-love** [comment]
yeah, unicode handling in R (under windows) is nothing short of a train-wreck.

i could describe a bewildering array of issues and wild unreproducible inconsistencies - particularly on non-english systems. these <U+....> things show up in the randomest places. even here:

https://www.rdocumentation.org/packages/jmv/versions/0.9.2.0/topics/contTables

i've been meaning to introduce an ascii mode.
